oresat_configs/edl_cmd_defs.py

```diff
@@ -0,0 +1,227 @@
+"""Used to parse edl_cmd_defs.yaml that defines all EDL command requests and responses."""
+
+import struct
+from dataclasses import dataclass, field
+from typing import Any, Union
+
+from dacite import from_dict
+from yaml import CLoader, load
+
+_COMMAND_DATA_FMT = {
```

Comment:
Similar to "magic numbers", are we using magic letters here? Or is this tied to a spec?

Reply:
These are [struct format characters](https://docs.python.org/3/library/struct.html#format-characters). You can see them being used down below in `struct.pack()` and `struct.unpack()` for [format strings](https://docs.python.org/3/library/struct.html#format-strings).